<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_OptMarginsBase</class>
 <widget class="QFrame" name="PropertyWidget_OptMarginsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QRadioButton" name="optMarginRadioNone">
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optMarginRadioBoth">
     <property name="text">
      <string>Both Sides</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optMarginRadioLeft">
     <property name="text">
      <string>Left Only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optMarginRadioRight">
     <property name="text">
      <string>Right Only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="optMarginResetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
